.github/workflows/smoke-tests-heroku.yml

```
@@ -0,0 +1,39 @@
+name: Smoke Tests Nightly -Heroku
+on:
+  schedule:
+    - cron:  '0 0 * * *'
```

let's test it against your PR - we can use the tars generated to help @TheSpartan1980 debug the issues.
```suggestion
on:
  schedule:
    - cron:  '0 0 * * *'
  push:
    branches: 
      - <name_of_your_branch>
```

---

Done. They are running, Some test failures.